mlflow/models/utils.py

```diff
@@ -264,3 +263,17 @@ def plot_lines(data_series, xlabel, ylabel, legend_loc=None, line_kwargs=None):
     ax.set(xlabel=xlabel, ylabel=ylabel)
 
     return fig, ax
+
+
+def create_wheeled_model(artifact_path, model_uri):
+    """
+    Given a registered model, this helper will re-log the model along with the wheels of all the
+    the model dependencies (stored along with the model artifacts).
+
+    :param artifact_path: Run-relative artifact path.
```

Comment:
Without additional context, this parameter docstring is unclear: "Why do I have to reason about a run-relative artifact path? I thought I was adding libraries to a model version in the MLflow model registry?" 

As I understand it, this API could be used to add dependencies to an existing model in MLflow Tracking without re-logging, right? In that case, would we be able to make `artifact_path` optional and infer it from the URI if the URI is ` runs:/` URI?